ably/rest_client_test.go

```diff
@@ -377,3 +377,45 @@ func TestRest_hostfallback(t *testing.T) {
 		}
 	})
 }
+
+func TestRestChannels(t *testing.T) {
```

Comment:
Generally you should try to have the spec "code" (like `RSN1`) somewhere in the test and/or subtest name, and channel names for each test should be tied to the test name somehow (so e.g. `rsn1_first_channel` or whatever in this case) so that parallel tests (which the library doesn't seem to currently use, but should probably be considered) don't interfere with each other